src/content/docs/concepts/security/runtime-authority.mdx

```diff
@@ -0,0 +1,12 @@
+---
+title: Runtime Authority
+---
+
+#### Runtime Authority
+
+The runtime authority is part of the Tauri core.
+It holds all permissions, capabilities and scopes at runtime to enforce
+which window can access which command and passes scopes to commands.
+
+
+![IPC Diagram](@assets/concepts/runtime-authority.svg)
```

Comment:
I think this needs an accompanying description.
This is a good diagram for people familiar with Tauri but difficult to understand for users who don't know Tauri-specific vocabulary